conf/defaults.ini

```diff
@@ -221,6 +221,9 @@ google_analytics_ua_id =
 # Google Analytics 4 tracking code, only enabled if you specify an id here
 google_analytics_4_id = 
 
+# When Google Analytics 4 Enhanced event measurement is enabled, we will try to avoid sending duplicate events and let Google Analytics 4 detect navigation changes, etc.
+google_analytics_4_send_manual_page_views = true
```

Comment:
if enhanced measurement defaults to true in GA, then do you think we should default to not sending page views?

Reply:
Correct, changed https://github.com/grafana/grafana/pull/57619/commits/4963aaaa72a57478eba5d1d6f01c232fafbec4cd